id: PYSEC-0000-CVE-2018-8976
severity:
- type: CVSS_V3
  score: CVSS:3.1/AV:N/AC:L/PR:N/UI:R/S:U/C:N/I:N/A:H
details: In Exiv2 0.26, jpgimage.cpp allows remote attackers to cause a denial of
  service (image.cpp Exiv2::Internal::stringFormat out-of-bounds read) via a crafted
  file.
affected:
- package:
    name: exiv2
    ecosystem: PyPI
    purl: pkg:pypi/exiv2
  ranges:
  - type: ECOSYSTEM
    events:
    - introduced: "0"
references:
- type: EVIDENCE
  url: https://github.com/Exiv2/exiv2/issues/246
- type: REPORT
  url: https://github.com/Exiv2/exiv2/issues/246
- type: FIX
  url: https://github.com/Exiv2/exiv2/issues/246
- type: ADVISORY
  url: https://security.gentoo.org/glsa/201811-14
- type: ADVISORY
  url: https://access.redhat.com/errata/RHSA-2019:2101
- type: ARTICLE
  url: https://lists.debian.org/debian-lts-announce/2023/01/msg00004.html
- type: WEB
  url: https://lists.debian.org/debian-lts-announce/2023/01/msg00004.html
aliases:
- CVE-2018-8976
modified: "2023-01-13T16:34:00Z"
published: "2018-03-25T03:29:00Z"
